frontend/src/scenes/organization/Settings/DangerZone.tsx

```diff
@@ -23,13 +23,14 @@ export function DeleteOrganizationModal({
         <Modal
             title="Delete the entire organization?"
             okText={`Delete ${currentOrganization ? currentOrganization.name : 'the current organization'}`}
-            okType="danger"
             onOk={currentOrganization ? () => deleteOrganization(currentOrganization) : undefined}
+            okType="primary"
```

Comment:
What action did intend for me to take on this? Is it just informative, or do you want me to support in some way? I have other button designs that I've been using for months, but we're not really using them consistently because our component reuse isn't great yet. Happy to support, just not sure what is needed here.